easybuild/easyconfigs/s/sradownloader/sradownloader-3.9-GCCcore-11.3.0.eb

```
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'sradownloader'
+version = '3.9'
+
+homepage = "https://github.com/s-andrews/sradownloader"
+description = """SRAdownloader takes the annotation table from the SRA run selector tool 
+and retrives the raw fastq files for the selected samples"""
```

```suggestion
and retrieves the raw fastq files for the selected samples"""
```
Yes, this should really be fixed upstream too, and apologies for being picky...